addons/account/models/reconciliation_widget.py

```diff
@@ -189,7 +214,7 @@ def get_bank_statement_line_data(self, st_line_ids, excluded_ids=None):
         return results
 
     @api.model
-    def get_bank_statement_data(self, bank_statement_line_ids, search_str=False):
+    def get_bank_statement_data(self, bank_statement_line_ids, srch_domain=[]):
```

Comment:
don't use an empty list as default parameter, this is a mutable object (cf https://llbit.se/?p=3440 )

Reply:
@mart-e How dare you commenting publicly on a closed PR? :)
Don't use an URL shortener (cf https://www.forbes.com/sites/ygrauer/2016/04/20/five-reasons-you-should-stop-shortening-urls/#5d0f6aed3f69)

I'll fix that.